### ISSUE TITLE
why dropouts are 0 for codegen-350M-mono?

### ISSUE THREAD

**Issue by wasiahmad** [salesforce/CodeGen#25]
Hi,

I noticed in the config file (https://huggingface.co/Salesforce/codegen-350M-mono/blob/main/config.json) that:

```
"attn_pdrop": 0.0
"embd_pdrop": 0.0
"resid_pdrop": 0.0
```

Is codegen pretrained with dropout 0? @enijkamp

**Comment by enijkamp** [comment]
Yes.

In training, the model does not include drop-out regularization, hence dropout of 0.0 in the converted PyTorch forward pass.

**Comment by wasiahmad** [comment]
@enijkamp What was the reason of not using any dropout? Want to learn if there is any insight. Thanks!
